src/patternfly/components/Progress/progress.scss

```diff
@@ -129,20 +124,20 @@
 
   // status modifiers
   &.pf-m-success {
-    --#{$progress}__bar--before--BackgroundColor: var(--#{$progress}--m-success__bar--BackgroundColor);
+    --#{$progress}__indicator--BackgroundColor: var(--#{$progress}--m-success__bar--BackgroundColor);
     --#{$progress}__status-icon--Color: var(--#{$progress}--m-success__status-icon--Color);
-    --#{$progress}--m-inside__measure--Color: var(--#{$progress}--m-success--m-inside__measure--Color);
```

Comment:
Looks like this should be added back, and the var pointing to the text--color--on-success token. Currently the color on the success indicator works, but it's using the on-brand token.